src/libraries/Pool.sol

```diff
@@ -441,6 +441,7 @@ library Pool {
             self.feeGrowthGlobal1X128 = state.feeGrowthGlobalX128;
         }
 
+        // in an unchecked block so sign flipping is not safe without a helper function
```

Comment:
is this comment needed? i think the helper got removed

Reply:
oh yeah it did good catch ty